mainapp/tests/test_models.py

```diff
@@ -0,0 +1,77 @@
+from django.test import TestCase
+# Create your tests here.
+from mainapp.models import Request, Volunteer, NGO, Contributor, DistrictManager, DistrictNeed, DistrictCollection, RescueCamp, Person
+
+class ModelTest(TestCase):
+	def test_request_string_representation(self):
+		request = Request(
+			district="pkd",
```

Comment:
Hey I believe you are trying to test the model layer but for that you must use either `request.save` or `Request.objects.create` else this would be just `Class` string representation test(if this is not what you intended to). Here there is no database hitting happening.